packages/ckeditor5-ui/src/editorui/accessibilityhelp.ts

```diff
@@ -0,0 +1,382 @@
+			options = {
+				keystrokes: options
+			};
+		} else if ( typeof options === 'object' && 'keystroke' in options ) {
+			options = {
+				keystrokes: [ options ]
+			};
+		}
+
+		const categoryId = options.category || DEFAULT_CATEGORY_ID;
+
+		if ( !this._keystrokes.has( categoryId ) ) {
+			/**
+			 * Cannot register keystrokes in an unknown category. Use
+			 * {@link module:ui/editorui/accessibilityhelp~AccessibilityHelp#registerKeystrokeCategory}
+			 * to register a new category or make sure the specified category exists.
+			 *
+			 * @error accessibility-help-unknown-category
+			 * @param categoryId The id of the unknown keystroke category.
+			 */
+			throw new CKEditorError( 'accessibility-help-unknown-category', { categoryId } );
+		}
+
+		const category = this._keystrokes.get( categoryId )!;
+		const groupId = options.group || DEFAULT_GROUP_ID;
+
+		if ( !category.groups.has( groupId ) && !options.groupLabel ) {
+			/**
+			 * Cannot register keystrokes in an unknown group.
+			 *
+			 * * If you want to create a new group, make sure you pass the `groupLabel` option to the
+			 * {@link module:ui/editorui/accessibilityhelp~AccessibilityHelp#registerKeystrokes} method.
+			 * * If you want to add keystrokes to an existing group, make sure it exists.
+			 *
+			 * @error accessibility-help-unknown-group
+			 * @param groupId The id of the unknown keystroke group.
+			 * @param categoryId The id of category the unknown group should belong to.
+			 */
+			throw new CKEditorError( 'accessibility-help-unknown-group', { groupId, categoryId } );
+		} else if ( options.groupLabel ) {
+			category.groups.set( groupId, {
+				label: options.groupLabel,
+				keystrokes: options.keystrokes
+			} );
+		} else {
+			category.groups.get( groupId )!.keystrokes.push( ...options.keystrokes );
+		}
+	}
+
+	/**
+	 * Injects a help text into each editing root's `aria-label` attribute allowing assistive technology users
+	 * to discover the availability of the Accessibility help dialog.
+	 */
+	private _setupRootLabels() {
+		const editor = this.editor;
+		const t = editor.t;
+
+		editor.on( 'ready', () => {
+			editor.editing.view.change( writer => {
+				for ( const rootName of editor.model.document.getRootNames() ) {
+					const viewRoot = editor.editing.view.document.getRoot( rootName );
+					const currentAriaLabel = viewRoot!.getAttribute( 'aria-label' );
+
+					writer.setAttribute(
+						'aria-label',
+						`${ currentAriaLabel }. ${ t( 'Press %0 for help.', [ getEnvKeystrokeText( 'Alt+0' ) ] ) }`,
+						viewRoot!
+					);
+				}
+			} );
+		} );
+	}
+
+	/**
+	 * Shows the accessibility help dialog. Also, creates {@link #contentView} on demand.
+	 */
+	private _showDialog() {
+		const editor = this.editor;
+		const dialog = editor.plugins.get( 'Dialog' );
+		const t = editor.locale.t;
+
+		if ( !this.contentView ) {
+			this.contentView = new AccessibilityHelpContentView( editor.locale, this._keystrokes );
+		}
+
+		dialog.show( {
```

Comment:
I believe dialog is of type `PluginInterface`. To prevent TS errors, we need to import types.

Reply:
WFM 
<img width="668" alt="image" src="https://github.com/ckeditor/ckeditor5/assets/1099479/4b612033-45c2-4cfd-92d7-15db3e401fbf">

Reply:
Weird...
<img width="389" alt="Zrzut ekranu 2024-01-18 o 16 37 46" src="https://github.com/ckeditor/ckeditor5/assets/20561161/80e398b3-c591-4c17-8b20-d7f85b585ac1">